integration/util.py

```diff
@@ -801,7 +866,7 @@ def __init__(self, path, timeout):
 
 
 
-@log_call(action_type=u"integration:await-file-contents", include_args=["path", "contents", "timeout"])
+@log_inline_callbacks(action_type=u"integration:await-file-contents", include_args=True)
```

Comment:
This was blocking and returned None.  Now it is reactor-driven and returns a Deferred.  Probably worth a docstring change.